common/selectors/selectAndDedupe.js

```diff
@@ -13,14 +13,12 @@ const dedupe = require("lib/dedupe");
 module.exports = function selectAndDedupe(group) {
   return group.reduce((result, options, index, arr) => {
     let current;
+    let sites = options.defaults ? options.sites.concat(options.defaults) : options.sites;
```

Comment:
Unfortunately because `option.defaults` is allowed to be `undefined` (for example, there are no defaults for Highlights) `options.sites.concat(options.defaults)` would result in `undefined` being added to the array, so I think the ternary is necessary unfortunately